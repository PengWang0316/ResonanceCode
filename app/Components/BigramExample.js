import React from 'react';

import Bigram46Example from '../imgs/Bigram46Example.png';
import Bigram13Example from '../imgs/Bigram13Example.png';

const imageStyle = { width: '100%' };

const BigramExample = props => (
  <div className="row">
    <div className="col-lg"><img style={imageStyle} src={Bigram46Example} alt="Bigram 46" /></div>
    <div className="col-lg"><img style={imageStyle} src={Bigram13Example} alt="Bigram 13" /></div>
  </div>
);
export default BigramExample;
