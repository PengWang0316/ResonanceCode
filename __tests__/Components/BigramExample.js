import React from 'react';
import renderer from 'react-test-renderer';
// import { shallow } from 'enzyme';

import BigramExample from '../../app/Components/BigramExample';

describe('BigramExample test', () => {
  const defaultProps = {};
  // const getShallowComponent = (props = defaultProps) => shallow(<BigramExample {...props} />);

  test('BigramExample snapshot', () => expect(renderer.create(<BigramExample {...defaultProps} />).toJSON()).toMatchSnapshot());
});
