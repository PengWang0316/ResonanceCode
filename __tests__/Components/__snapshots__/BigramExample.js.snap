// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BigramExample test BigramExample snapshot 1`] = `
<div
  className="row"
>
  <div
    className="col-lg"
  >
    <img
      alt="Bigram 46"
      src={
        Object {
          "process": [Function],
        }
      }
      style={
        Object {
          "width": "100%",
        }
      }
    />
  </div>
  <div
    className="col-lg"
  >
    <img
      alt="Bigram 13"
      src={
        Object {
          "process": [Function],
        }
      }
      style={
        Object {
          "width": "100%",
        }
      }
    />
  </div>
</div>
`;
